### PR TITLE
Minor refactor

### PR DESCRIPTION
**Pull request by justin1121**
- Noticed a random source was being called salt which I don't think it technically is
- Created a const for the AES key size.
- Minor code moving around

### CODE REVIEW

**Comment by shwetasah-cape** [comment]
where's the test file?

**Comment by justin1121** [comment]
Forgot the add it. Thanks!

File: sdk/encrypt_test.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"path"
+	"testing"
+)
+
+func TestEncrpt(t *testing.T) {
```
Comment (annieltan) [inline sdk/encrypt_test.go:17]:
```suggestion
func TestEncrypt(t *testing.T) {
```

**Review by shwetasah-cape — APPROVED** [review]
looks good to me!

**Pull request merged**